src/main/java/io/conduit/DestinationConfig.java

```diff
@@ -66,6 +66,9 @@ public class DestinationConfig {
     @JsonProperty(value = "s3.secret-access-key")
     private String s3SecretAccessKey;
 
+    @JsonProperty(value = "s3.region")
```

Comment:
so do we need to add this to the specifier too?

Reply:
Oh, right!

Reply:
so if our destination is minIO, we'd just leave this param empty?

Reply:
oh I see! makes sense, thanks